src/demo-app/icon/icon-demo.scss

```diff
@@ -0,0 +1,7 @@
+md-icon {
+  color: purple;
+}
+
+md-icon.green {
+  color: green;
+}
```

Comment:
Can you prefix all of the demo styles with `.demo-`? This makes it easy for people to inspect the elements and immediately see which styles are from the component and which are from the demo.